### ISSUE TITLE
double tabindex in <select>

### ISSUE THREAD

**Issue by rubenscd** [Dogfalo/materialize#5512]
When asking general "how to" questions:

- Please do not open an issue here!
- Instead, ask for help on [StackOverflow](stackoverflow.com/questions/tagged/materialize) or in our [Gitter](https://gitter.im/Dogfalo/materialize) channel.

- Read the [CONTRIBUTING document](https://github.com/Dogfalo/materialize/blob/master/CONTRIBUTING.md) Any issues that are violating the contributing guidelines, will be closed!
- [Search for duplicate or closed issues](https://github.com/Dogfalo/materialize/issues?utf8=%E2%9C%93&q=is%3Aissue) and make sure to go through our [labels](https://github.com/Dogfalo/materialize/labels).
- Prepare a [reduced test case](https://css-tricks.com/reduced-test-cases/) for any bugs.

## Expected Behavior

please add dynamic `tabindex="-1"` to generate materialize `<select>` to avoid double TAB 

## Current Behavior

all materialize `<select>` create dynamic <input> in parallel, forcing **the double** tab in `<select>` 

## Steps to Reproduce (for bugs)

1. https://codepen.io/rubenscd/pen/rpwdYm
2. tabbing in keybord first input
3. ...

## Your Environment

* Version used: 1.0.0-alpha.3 or 0.100.2
* Browser Name and version: all
* Operating System and version (desktop or mobile): macos 10.13.2
* Link to your project (if appropriate): https://codepen.io/rubenscd/pen/rpwdYm

**Comment by erichiggins** [comment]
I just ran into this issue as well, but I did find a possible short-term solution.

Simply set the `tabindex=-1` on your `select` element.  The original `select` element is hidden and replaced by an `input` element, and that attribute is *not* copied over to the generated `input`.

**One caveat:** If you have multiple `select` fields in your form, you'll need to increment your `tabindex` value (e.g. `tabindex=-1`, `tabindex=-2`, `tabindex=-3`), otherwise only the first one will work as expected.
